pkg/util/crypt/crypt_dev.go

```diff
@@ -143,7 +147,7 @@ func (crypt *Device) EncryptFilesystem(path string, key []byte) (string, error)
 	sylog.Debugf("Running %s %s", cmd.Path, strings.Join(cmd.Args, " "))
 	err = cmd.Run()
```

Comment:
Alternatively, we can use cmd.CombinedOutput() and print the command output in case of an error.

Reply:
I would prefer no to because separating the two is actually extremely useful in the context of an error.